Dockerfile

```diff
@@ -65,32 +61,38 @@ RUN apt-get update && apt-get install -y --no-install-recommends  \
     vim                   \
     wget                  \
     zip                   \
+    libpq-dev             \
   && rm -rf /var/lib/apt/lists/* \
   && apt-get clean all
 
 # Install conda.
 RUN cd /tmp && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
-    echo "${MINICONDA_SHA256} *Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh" | sha256sum -c - && \
-    /bin/bash Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
-    rm Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh && \
-    echo "conda ${CONDA_VERSION}" >> $CONDA_DIR/conda-meta/pinned && \
-    conda config --system --prepend channels conda-forge && \
-    conda config --system --set auto_update_conda false && \
-    conda config --system --set show_channel_urls true && \
-    conda list python | grep '^python ' | tr -s ' ' | cut -d '.' -f 1,2 | sed 's/$/.*/' >> $CONDA_DIR/conda-meta/pinned && \
-    conda install --quiet --yes conda && \
-    conda install --quiet --yes pip && \
-    conda update --all --quiet --yes && \
+    export ARCH=`uname -m` && \
+    if [ "$ARCH" = "x86_64" ]; then \
+       echo "x86_64" && \
+       export MINICONDA_ARCH=x86_64 && \
+       export MINICONDA_SHA256=935d72deb16e42739d69644977290395561b7a6db059b316958d97939e9bdf3d; \
+    elif [ "$ARCH" = "aarch64" ]; then \
+       echo "aarch64" && \
+       export MINICONDA_ARCH=aarch64 && \
+       export MINICONDA_SHA256=19584b4fb5c0656e0cf9de72aaa0b0a7991fbd6f1254d12e2119048c9a47e5cc; \
+    else \
+       echo "unknown arch"; \
+    fi && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-${MINICONDA_ARCH}.sh && \
+    echo "${MINICONDA_SHA256} *Miniconda3-${MINICONDA_VERSION}-Linux-${MINICONDA_ARCH}.sh" | sha256sum -c - && \
+    /bin/bash Miniconda3-${MINICONDA_VERSION}-Linux-${MINICONDA_ARCH}.sh -f -b -p $CONDA_DIR && \
+    rm Miniconda3-${MINICONDA_VERSION}-Linux-${MINICONDA_ARCH}.sh && \
+    echo "conda ${CONDA_VERSION}" >> $CONDA_DIR/conda-meta/pinned  && \
+    conda config --system --prepend channels conda-forge  && \
+    conda config --system --set auto_update_conda false  && \
+    conda config --system --set show_channel_urls true  && \
+    conda list python | grep '^python ' | tr -s ' ' | cut -d '.' -f 1,2 | sed 's/$/.*/' >> $CONDA_DIR/conda-meta/pinned  && \
+    conda install --quiet --yes conda  && \
+    conda install --quiet --yes pip  && \
+    conda update --all --quiet --yes  && \
     conda clean --all -f -y
 
-# Upgrade ruamel.py version. Fixes https://github.com/aiidateam/aiida-core/issues/4339.
-# Note: miniconda 4.10.3 installs 0.15.80 by default
-RUN conda install ruamel.yaml==0.16.10
```

Comment:
@unkcpz are you sure that the issue https://github.com/aiidateam/aiida-core/issues/4339 doesn't pop up again if you delete this?

Reply:
Yes, I didn't see the issue. Maybe I didn't run `verdi shell` as described? I need have a double check, and check the actual version installed. I think I build the aiida-core 1.6.8 so it has already fixed the dependency. I'll check it now anyway.

Reply:
thanks!

Reply:
@yakutovicha, the issue is not there anymore. Should be safe to delete it.  cheers!